File: backend/app/models/Sala.php
<?php  defined('INITIALIZED') OR exit('You cannot access this file directly');

class Sala extends Model {
	
	private $id;
	private $codigo;
	private $bloco;
	private $tipoSala;
	private $administrador;
	private $equipamentos;
	private $ativo;

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getCodigo()
    {
        return $this->codigo;
    }

    public function setCodigo($codigo)
    {
        $this->codigo = $codigo;
    }

    public function getBloco()
    {
        return $this->bloco;
    }

    public function setBloco($bloco)
    {
        $this->bloco = $bloco;
    }

    public function getTipoSala()
    {
        return $this->tipoSala;
    }

    public function setTipoSala($tipoSala)
    {
        $this->tipoSala = $tipoSala;
    }

    public function getAdministrador()
    {
        return $this->administrador;
    }

    public function setAdministrador($administrador)
    {
        $this->administrador = $administrador;
    }

    public function getEquipamentos()
    {
        return $this->equipamentos;
    }

    public function setEquipamentos($equipamentos)
    {
        $this->equipamentos = $equipamentos;
    }

    public function getAtivo()
    {
        return $this->ativo;
    }

    public function setAtivo($ativo)
    {
        $this->ativo = $ativo;
    }
	
	
}

File: Lixo/frontend/src/models/Sala.java
<?php

defined('INITIALIZED') OR exit('You cannot access this file directly');

class Sala extends Model {

    private $id;
    private $codigo;
    private $idBloco;
    private $idTipoSala;
    private $idAdministrador;
    private $equipamentos;
    private $ativo;

    function getId() {
        return $this->id;
    }

    function getCodigo() {
        return $this->codigo;
    }

    function getIdBloco() {
        return $this->idBloco;
    }

    function getIdTipoSala() {
        return $this->idTipoSala;
    }

    function getIdAdministrador() {
        return $this->idAdministrador;
    }

    function getEquipamentos() {
        return $this->equipamentos;
    }

    function getAtivo() {
        return $this->ativo;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setCodigo($codigo) {
        $this->codigo = $codigo;
    }

    function setIdBloco($idBloco) {
        $this->idBloco = $idBloco;
    }

    function setIdTipoSala($idTipoSala) {
        $this->idTipoSala = $idTipoSala;
    }

    function setIdAdministrador($idAdministrador) {
        $this->idAdministrador = $idAdministrador;
    }

    function setEquipamentos($equipamentos) {
        $this->equipamentos = $equipamentos;
    }

    function setAtivo($ativo) {
        $this->ativo = $ativo;
    }

}
